webdriver/status.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+
+def assert_status_response(http_status, res):
+        assert http_status == 200
+        assert res["ready"] in [True, False]
+        assert isinstance(res["message"], basestring)
+
+
+# TODO: figure out if geckodriver supports sessionless status
+# currently response body for GET /status without a session is
+# coming back empty
+def test_get_status_no_session(http):
+    # with http.get("status", use_json=True) as result:
+        # [status, obj] = result
+        # assert_status_response(status, obj)
+    pass
+
+
+def test_get_status_with_session(session):
```

Comment:
This test makes a request for a resource that isn't defined by the WebDriver specification. Could you remove it?